Resources/application.py

```diff
@@ -100,26 +101,39 @@ def __init__(self, customDatetime):
         self.connectEventsDensityTab()
 
         self.threadPool = QtCore.QThreadPool()
+        self.writeConfig('savefilename','')
 
         return
 
+    def readConfig(self, configKey, configGroup='DEFAULT'):
```

Comment:
I'd just import this function from Resources/Functions/miscFunctions instead of re-writing the whole function here.